client/my-sites/site-settings/theme-enhancements/index.jsx

```diff
@@ -139,27 +135,28 @@ class ThemeEnhancements extends Component {
 					disabled={ formPending }
 					/>
 
-				{
-					minilevenModuleActive && (
-						<div className="theme-enhancements__module-settings is-indented">
-							{
-								this.renderToggle( 'wp_mobile_excerpt', translate(
-									'Show excerpts on front page and on archive pages instead of full posts'
-								) )
-							}
-							{
-								this.renderToggle( 'wp_mobile_featured_images', translate(
-									'Hide all featured images'
-								) )
-							}
+				<div className="theme-enhancements__module-settings is-indented">
+					{
+						this.renderToggle( 'wp_mobile_excerpt', ! minilevenModuleActive, translate(
+							'Show excerpts on front page and on archive pages instead of full posts'
+						) )
+					}
+					{
+						this.renderToggle( 'wp_mobile_featured_images', ! minilevenModuleActive, translate(
+							'Hide all featured images'
+						) )
+					}
+					{
+						this.renderToggle( 'wp_mobile_app_promos', ! minilevenModuleActive, translate(
+							'Show an ad for the {{link}}WordPress mobile apps{{/link}} in the footer of the mobile theme',
```

Comment:
Alternate string suggestion: 
* `translate( 'Show an ad for the WordPress mobile apps in the footer of the mobile theme' )`&mdash; translations: **1**. *ES Score: 5.74*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/site-settings-theme-enhancements-hidden-to-disabled)